packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -404,95 +404,162 @@ const AccountTransferForm = observer(
-                                            is_large
-                                            label={localize('From')}
-                                            list={from_accounts}
-                                            list_height='404'
-                                            name='transfer_from'
-                                            value={selected_from.value}
-                                            onChange={(e: TReactChangeEvent) => {
-                                                onChangeTransferFrom(e);
-                                                handleChange(e);
-                                                setFieldValue('amount', '');
-                                                setTimeout(() => setFieldError('amount', ''));
-                                            }}
-                                            error={selected_from.error}
-                                        />
-                                        <Dropdown
-                                            id='transfer_to'
-                                            className='account-transfer-form__drop-down account-transfer-form__drop-down--to-dropdown'
-                                            classNameDisplay='cashier__drop-down-display'
-                                            classNameDisplaySpan='cashier__drop-down-display-span'
-                                            classNameItems='cashier__drop-down-items'
-                                            classNameLabel='cashier__drop-down-label'
-                                            classNameHint='account-transfer-form__hint'
-                                            test_id='dt_account_transfer_form_to_dropdown'
-                                            is_large
-                                            label={localize('To')}
-                                            list={to_accounts}
-                                            list_height='404'
-                                            name='transfer_to'
-                                            value={selected_to.value}
-                                            onChange={(e: TReactChangeEvent) => {
-                                                onChangeTransferTo(e);
-                                                setFieldValue('amount', '');
-                                                setTimeout(() => setFieldError('amount', ''));
-                                            }}
-                                            hint={transfer_to_hint}
-                                            error={getMt5Error() ?? selected_to.error}
-                                        />
+                                <>
+                                    <div className='account-transfer-form__inline-warning-message'>
+                                        {!Number(remaining_transfers) && (
```

Comment:
@farhan-nurzi-deriv `!Number(remaining_transfers)` has been used 6 times, Can we move it to a named variable 🙏🏻 

```ts
const has_reached_maximum_daily_transfers = !Number(remaining_transfers);
```

Reply:
Sure 🙏